ReleaseNotes_Viewer.txt

```diff
@@ -15,6 +15,94 @@ Known issues
 Changes
 -------
 
+2.28.0
+
+* Configure Keyboard Shortcuts (#637)
```

Comment:
Still the same hint: This is obsolete, because release job generates this automatically

Reply:
I made some cleanup here.